doc/doxygen_templates/pages.txt

```diff
@@ -0,0 +1,105 @@
+/**
+@mainpage Overview
+@brief [<FIXME: library description>]
+
+[<FIXME: Some useful overview text>]
+
+@section [<FIXME: Lowercase library name>]_memory_requirements Memory Requirements
+@brief Memory requirements of the [<FIXME: Library name>] library.
+<table>
+    <tr>
+        <td colspan="3"><b>Code Size of [<FIXME: Library name and version>](example generated with GCC for ARM Cortex-M)</b></td>
+    </tr>
+    <tr>
+        <td><b>File</b></td>
+        <td><b>With -O1 Optimisation</b></td>
+        <td><b>With -Os Optimisation</b></td>
+    </tr>
+    <tr>
+        <td>[<FIXME: library source, add more rows for more source>]</td>
+        <td>[<FIXME: Code size>]</td>
+        <td>[<FIXME: Code size>]</td>
+    </tr>
+    <tr>
+        <td><b>Total estimate (configurations defined <a href="https://github.com/FreeRTOS/FreeRTOS/tree/lts-development/tools/memory_estimator" target="_blank" rel="noopener noreferrer">here</a>)</b></td>
```

Comment:
I think mqtt changed this table a bit before it was merged, so you could make the same changes here too

Reply:
Sure thanks